packages/react/src/breadcrumbs/docs/overview.mdx

```diff
@@ -20,9 +20,9 @@ Breadcrumbs show users where they are in the website hierarchy and how to naviga
 
 <DoHeading />
 
-- display above the current page title (h1)
-- display the current page title in the breadcrumbs
-- match the breadcrumb to full page title
+- display above the main page heading (h1) and before the `main` in the code
```

Comment:
I think this should be 
'...before 'main' in the code' (remove 'the' ), or
'...before the 'main' section in the code'

Reply:
Looks good to me. Marking this as resolved.